src/main/java/sx/blah/discord/handle/obj/IEmbedded.java

```diff
@@ -0,0 +1,71 @@
+package sx.blah.discord.handle.obj;
+
+import sx.blah.discord.handle.impl.obj.Embedded;
+
+/**
+ * Represents media embedded in the message.
+ */
+
+public interface IEmbedded {
```

Comment:
This should still be a subclass of IMessage because it is only used in the context of a message